src/debug.c

```diff
@@ -4538,4 +4980,4 @@ void CheckEWRAMCounters(struct ScriptContext *ctx)
 static void DebugAction_Util_CheckEWRAMCounters(u8 taskId)
 {
     Debug_DestroyMenu_Full_Script(taskId, Debug_EventScript_EWRAMCounters);
-}
+}
```

Comment:
Missing newline

Reply:
You added a newline before the brace, but it needs to be at the end of the file :)